packages/glslls/package.yaml

```
@@ -0,0 +1,32 @@
+---
+name: glslls
+description: Language server implementation for GLSL.
+homepage: https://github.com/svenstaro/glsl-language-server
+licenses:
+  - MIT
+languages:
+  - GLSL
+categories:
+  - LSP
+
+source:
+  # renovate:datasource=git-refs
+  id: pkg:github/svenstaro/glsl-language-server@92de13c10a4855e935e6cd2625217ec040b4767a
```

They seem to tag releases - let's target that instead.

```suggestion
  # renovate:datasource=github-tags
  id: pkg:github/svenstaro/glsl-language-server@0.5.0
```